tools/export/makefile/__init__.py

```diff
@@ -288,18 +288,21 @@ class Armc5(Arm):
     @classmethod
     def is_target_supported(cls, target_name):
         target = TARGET_MAP[target_name]
-        if int(target.build_tools_metadata["version"]) > 0:
-            #Although toolchain name is set to ARM above we should check for ARMC5 for 5.12/onwards
-            if "ARMC5" not in target.supported_toolchains:
-                return False
-
-        arm_res = apply_supported_whitelist(
-            "ARM", cls.POST_BINARY_WHITELIST, target
-        )
-        armc5_res = apply_supported_whitelist(
-            "ARMC5", cls.POST_BINARY_WHITELIST, target
-        )
-        return arm_res or armc5_res
+        if not target.is_TFM_target:
+            if int(target.build_tools_metadata["version"]) > 0:
+                #Although toolchain name is set to ARM above we should check for ARMC5 for 5.12/onwards
```

Comment:
Would recommend adding spaces after `#` in Python comments, in a separate commit from the comimt that wraps this in `if not target.is_TFM_target`

Reply:
Fixed